### PR TITLE
docs: azure ad b2c

### PR DESCRIPTION
**Pull request by chirdeeptomar**
Adding documentation for authenticating with Azure AD B2C

Issue: https://github.com/refinedev/refine/issues/2902

### CODE REVIEW

**Comment by aliemir** [comment]
Hey @chirdeeptomar looks fantastic 🪄 Thank you for the tutorial 👏 

About your changes in the contributing.md file, we've updated it recently and its deployed. We also added a mention for the commit conventions and the process. You can check the changes here https://refine.dev/docs/contributing/#commit-convention - I think your changes to the file might not be necessary with the latest changes 🤔

**Comment by omeraplak** [comment]
Hey @chirdeeptomar ,
Thank you for your contribution! We'll review it soon!

**Comment by omeraplak** [comment]
Hey @chirdeeptomar ,
I made a few updates by following our Auth0 document. Can you review the changes?

File: documentation/docs/advanced-tutorials/auth/azure-ad.md [hunk 1]
```diff
@@ -84,58 +126,67 @@ root.render(
 );
 ```
 
-config.tsx
 
-```typescript
-import { Configuration, LogLevel } from "@azure/msal-browser";
+## Override login page
+First, we need to override the refine login page. In this way, we will redirect it to the Auth0 login page. We create a `login.tsx` file in the `/src` folder.
```
Comment (chirdeeptomar) [inline documentation/docs/advanced-tutorials/auth/azure-ad.md:131]: Replace Auth0 with Azure AD

Comment (omeraplak) [reply documentation/docs/advanced-tutorials/auth/azure-ad.md]: Thank you! I've fixed

**Pull request merged**